docs/installation/osx.md

```diff
@@ -26,7 +26,7 @@ Type `psql` into command line. Then you should see this:
 Next to that type `ALTER ROLE "postgres" WITH CREATEDB`.  (You may need to create this role, see [issues](issues.md) for help.)
 1. `git checkout develop`.
 1. Run `bundle install` to get the gems.
-1. Run `npm install` to get the javascript dependencies.
+1. Run `yarn install` to get the javascript dependencies.
```

Comment:
Change this back to `npm` so we use `package-lock.json` to lock package versions